rest_framework/pagination.py

```diff
@@ -36,10 +36,14 @@ def _positive_int(integer_string, strict=False, cutoff=None):
 
 def _divide_with_ceil(a, b):
     """
-    Returns 'a' divded by 'b', with any remainder rounded up.
+    Returns 'a' divided by 'b', with any remainder rounded up.
     """
+    if a == 0 or b == 0:
```

Comment:
@tomchristie this seems to just be used in `get_html_context()`. I can just check if `self.limit` is 0 there and take care of things without changing `_divide_with_ceil`.